src/lib/select/select.spec.ts

```diff
@@ -62,3203 +70,3284 @@ describe('MatSelect', () => {
   let scrolledSubject = new Subject();
   let viewportRuler: ViewportRuler;
 
-  beforeEach(fakeAsync(() => {
+  // Providers used for all mat-select tests
+  const commonProviders = [
+    {
+      provide: OverlayContainer, useFactory: () => {
+      overlayContainerElement = document.createElement('div') as HTMLElement;
```

Comment:
I might have it in another PR somewhere, but we shouldn't be creating thing element in the first place, we should just inject the `OverlayContainer`.

Reply:
I'd rather address that in a different PR because I don't want to actually change any logic in this one